insights/config/specs.py

```diff
@@ -398,6 +398,8 @@
     "sshd_config_perms"         : CommandSpec("/bin/ls -l /etc/ssh/sshd_config"),
     "sssd_config"               : SimpleFileSpec("etc/sssd/sssd.conf"),
     "sssd_logs"                 : PatternSpec(r"var/log/sssd/.*\.log$"),
+    "subscription_manager_list_consumed": CommandSpec("subscription-manager list --consumed"),
+    "subscription_manager_list_installed": CommandSpec("subscription-manager list --installed"),
```

Comment:
@PaulWay you'll need to add the path to this command.

Reply:
That's stupid of me, especially since I wrote the code that checked for that!  Oops!  I've fixed it in this update.  Thanks @bfahr